apis/s3/v1alpha2/bucketpolicy_types.go

```diff
@@ -24,7 +24,6 @@ import (
 
 // BucketPolicyParameters define the desired state of an AWS BucketPolicy.
 type BucketPolicyParameters struct {
-
 	// Region is where the Bucket referenced by this BucketPolicy resides.
 	Region string `json:"region"`
 
```

Comment:
I think `PolicyStatement []BucketPolicyStatement json:"statement"` should be `Statements []BucketPolicyStatement json:"statements"`